packages/block-library/src/table/editor.scss

```diff
@@ -3,9 +3,14 @@
 	&[data-align="right"],
 	&[data-align="center"] {
 		table {
+			display: table;
```

Comment:
@tellthemachines I think these table changes are the ones from #16250, right? If so, they should be removed from this PR to focus on just the sticky warning fixes. Thank you!

Reply:
Whoops, not sure how that happened! Will revert.

Reply:
Done.

Reply:
Thank you!